bin/dev

```diff
@@ -28,7 +28,7 @@ USER_ID_GROUP = %w[u g].map { |f| `id -#{f}`.chomp }.join(':')
 RUBY_IMAGE = options.fetch(:image, 'ruby:latest')
 FRAMEWORKS = options.fetch(:frameworks, 'rails,sinatra,grape')
 
-IMAGE_PATH_SAFE = RUBY_IMAGE.gsub(':', '_')
+IMAGE_PATH_SAFE = RUBY_IMAGE.tr(':', '_')
```

Comment:
Why did you change this?

Reply:
`rubocop-performance` suggests that it's faster than `gsub` for regular string (non-regex)

Reply:
oh nice, I didn't know that it was faster.